mediacloud/mediawords/db/schema/schema.py

```diff
@@ -32,7 +33,7 @@ def reset_all_schemas(db_: DatabaseHandler) -> None:
 
     label = decode_object_from_bytes_if_needed(label)
 
-    db = connect_to_db(label=label, do_not_check_schema_version=True)
+    db = mediawords.db.connect_to_db(label=label, do_not_check_schema_version=True)
```

Comment:
I get this when I try to import the connect_to_db name:
```
mediacloud/mediawords/tm/test_fetch_topic_tweets.py:9: in <module>
    from mediawords.db import DatabaseHandler
mediacloud/mediawords/db/__init__.py:2: in <module>
    from mediawords.test.db.env import using_test_database
mediacloud/mediawords/test/db/__init__.py:5: in <module>
    from mediawords.db.schema.schema import recreate_db
mediacloud/mediawords/db/schema/schema.py:1: in <module>
    from mediawords.db import connect_to_db
E   ImportError: cannot import name 'connect_to_db'
```

I started getting this problem when I tried to import mediawords.db into mediawords.test.db, I think because python is having some sort of weird circular dependence problem.  

I've reverted this after removing the test_on_test_database stuff as suggested below.